README.md

```diff
@@ -157,6 +157,25 @@ results = SequenceQuery("MTEYKLVVVGAGGVGKSALTIQLIQNHFVDEYDPTIEDSYRKQVVIDGET" +
 for polyid in results("polymer_entity"):
     print(polyid)
 ```
+### Sequence Motif Search Example
```

Comment:
This example looks great! Nice idea following from the example in the search API documentation—that's great consistency.  

If possible, I think it'd also be very nice to have examples for the other two pattern types and sequence types in the documentation as well, since in large part it would help advertise to the user the set of options that we have available (these are relatively newer search features, so not everybody is fully aware of their potential yet). I shared this document earlier, but feel welcome to peek at the examples at the bottom here for some ideas: https://www.rcsb.org/docs/search-and-browse/advanced-search/sequence-motif-search